tests/test_messaging.py

```diff
@@ -474,19 +474,91 @@ def test_no_body_loc_key(self):
         assert str(excinfo.value) == expected
 
     @pytest.mark.parametrize('data', NON_STRING_ARGS)
-    def test_invalid_channek_id(self, data):
+    def test_invalid_channel_id(self, data):
         notification = messaging.AndroidNotification(channel_id=data)
         excinfo = self._check_notification(notification)
         assert str(excinfo.value) == 'AndroidNotification.channel_id must be a string.'
 
+    @pytest.mark.parametrize('data', NON_STRING_ARGS)
```

Comment:
NON_STRING_ARGS is probably not the right choice here. May be just use an inlined list of invalid arguments for clarity.